Main/src/Main/Coffee_Machine.java

```diff
@@ -1,14 +1,31 @@
 package Main;
+import java.util.Scanner;
 
 public class Coffee_Machine {
-    public static void main(String[] args) {
-        System.out.println("Starting to make a coffee");
-        System.out.println("Grinding coffee beans");
-        System.out.println("Boiling water");
-        System.out.println("Mixing boiled water with crushed coffee beans");
-        System.out.println("Pouring coffee into the cup");
-        System.out.println("Pouring some milk into the cup");
-        System.out.println("Coffee is ready!");
 
-    }
-}
+    public static void main(String[] args) {
+        setIngredients(getCups());
```

Comment:
Add comments

Reply:
I created 3 method
getCups, to read input from user,
setIngredients to calculate necessary volume of ingredient and
getIngredients do printout ingredient's volume.
In main I call setIngredients, which receives getCup's result as input.
In the setIngredients I call getIngridients to printout the volume of ingredients,